installation/sql/mysql/extensions.sql

```diff
@@ -957,21 +956,21 @@ CREATE TABLE IF NOT EXISTS `#__guidedtour_steps` (
 --
 
 INSERT INTO `#__guidedtour_steps` (`id`, `tour_id`, `title`, `alias`, `published`, `description`, `ordering`, `step-no`, `position`, `target`, `url`, `multipage`, `created`, `created_by`, `modified`, `modified_by`, `state`) VALUES
```

Comment:
```suggestion
INSERT INTO `#__guidedtour_steps` (`id`, `tour_id`, `title`, `alias`, `published`, `description`, `ordering`, `step-no`, `position`, `target`, `url`, `created`, `created_by`, `modified`, `modified_by`, `state`) VALUES
```
If you remove it from the values then you also have to remove it from the columns, otherwise you get an SQL error.